payment-service/todo.md

```
@@ -22,8 +22,20 @@
 7. ✅커스텀 이벤트, 이벤트 핸들러 붙이기
 8. ✅주문 취소 이벤트를 소비하여 결제 취소 구현
 9. ✅카프카 리스너 공통 라이브러리로 리팩토링
-10. api-gateway-route
-
+10. ☑️주문 취소 이벤트 메시지 예시
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)